setup_python.sh

```diff
@@ -9,9 +9,11 @@ function detect_distro(){
 
 case $DD_TARGET_ARCH in
 "x64")
+    DD_CONDA_VERSION=4.9.2
     CONDA_URL=https://repo.anaconda.com/miniconda/Miniconda3-${DD_CONDA_VERSION}-Linux-x86_64.sh
```

Comment:
I think we have to do this?
```suggestion
    CONDA_URL=https://repo.anaconda.com/miniconda/Miniconda3-py39_${DD_CONDA_VERSION}-Linux-x86_64.sh
```